docs/operator-docs/common-use-cases.md

```diff
@@ -0,0 +1,15 @@
+---
+id: common-use-cases
+title: Common Use Cases
+hide_title: true
+---
+
+# Common Use Cases
+
+Listed below are some example use cases that you may encounter during your daily activities.
+
+* [AWS Long Term Access Key](../operator-docs/aws-long-term-access-key)
```

Comment:
This should stay in the wiki since it's NGAP specific

Reply:
@laurenfrederick Moved to wiki and removed from these docs.